### DISCUSSION TITLE
Module Style's in Astroid Templates

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#287] [Q&A]
Hello,
is there an easy way to format modules like this?
So with background, frame etc.?
![image](https://user-images.githubusercontent.com/19833064/217521694-b3fafd1e-565f-483b-8c51-942c8df5e3b1.png)

I do not mean the module position in the Layout Manager, but the individual modules.
In JSN this was implemented with Module Class Suffix and Module Style in the individual modules:
![image](https://user-images.githubusercontent.com/19833064/217522357-c8b403e5-a587-43e8-983c-cc3f4038e794.png)

WBR,
deltapapa

**Comment by deltapapa01** [comment]
Hi,
ok, I found it, in astroid there are also modules chrome. Can I put custom style in the file /templates/astroid_template_zero/html/modules.php, or this file will be overwritten during an update?

**Comment by Chacapamac** [comment]
I @deltapapa01 ;
I can see in both templates **Zero** and **One** the  **/templates/astroid_template_zero or one/html/modules.php** file.
Meaning, Yes you should be able to Override there.

Good Day to All.

The only thing weird, I cannot see the way to use Joomla administration override system to override “**modules.php**” ????
Maybe it’s to early in the morning for me?

**Comment by deltapapa01** [comment]
Hi @Chacapamac,
in Joomla 4 the modules chromes use a different way to override the module appearance:
Under /templates/astroid_template_zero/html/layouts/chromes you can find the different layout for the module style.
Here I copied one, changed and adjusted the name, created my own css rule and my own style was ready:

![image](https://user-images.githubusercontent.com/19833064/218018233-f57992e0-c2e4-470e-b544-f19900d78981.png)

Works wonderfully for me.
I think the modules.php is no longer relevant for Joomla 4, and only exists for compatibility with Joomla 3


WBR,
deltapapa

**Reply by Chacapamac** [reply]
Thanks @deltapapa01, I see, looking in **/templates/astroid_template_one/html/layouts/chromes** I’m seeing these files;
![Screen Shot 2023-02-10 at 10 10 50 AM](https://user-images.githubusercontent.com/1907276/218126535-4ac529e4-323d-40c6-897d-45e50591848d.jpg)

**Questions:**
**• I look the settings in a custom module and I was expecting to see a way to change the “Chrome” style?** 
**• How do you used the chromes and do you simply add a chromes in **/templates/astroid_template_one/html/layouts/chromes****
![Screen Shot 2023-02-10 at 10 13 43 AM](https://user-images.githubusercontent.com/1907276/218128542-b48fdc1b-1ee3-40ca-a5b1-333e78cf3e26.jpg)

**Reply by Chacapamac** [reply]
@deltapapa01 , question, when you create your override are you adding the module layout class manually?
— See my previous post.

**Comment by Chacapamac** [comment]
Thanks @deltapapa01 , I finally get it.
**I was looking at the wrong Module settings** in “**Module**” ➔ ”**Advanced**” tabs ➔ ”**Layouts**”, as it make sens to me :(
— **This settings is still mysterious to me?????** — Look at the choices when my test website is on **Template One**
![Screen Shot 2023-04-30 at 11 38 38 AM](https://user-images.githubusercontent.com/1907276/235362665-9c5407ae-74fa-4758-a8ee-0cac891693d3.jpg)

**• For other people watching that (and for the documentation) the right Module Settings is here:**
![Screen Shot 2023-04-30 at 11 48 17 AM](https://user-images.githubusercontent.com/1907276/235362775-933c5168-8714-40da-9ccd-ed8ebd509fe5.jpg)

and the files are under /templates/your_template/html/layouts/chromes you can find the different layout for the module style and create your own.
**I will add this to the documentation**

Good Day to All!
